lib/xenops_server.ml

```diff
@@ -947,6 +947,7 @@ let rec atomics_of_operation = function
     ) @ (List.map (fun vusb -> VUSB_unplug vusb.Vusb.id)
            (VUSB_DB.vusbs id)
         ) @ simplify ([
+        VM_pause id;
```

Comment:
This is _after_ the shutdown, as the below comment implies. So how does it help?

Reply:
Shutdown is cooperative. This is for the case where the domain is still running but we want to destroy the device model - this is how I understand it.

Reply:
Feels like a good explanation to add as a comment :)

Reply:
In a hard shutdown or snapshot revert `timeout` is `None` so the VM_shutdown_domain is not called. Therefore we need to either pause or destroy it here

Reply:
Yes, please update the comment to avoid confusion: the pause is only required in case of a hard shutdown. And also fix the comment below here, which is only true for a clean shutdown.